app/api/utils.py

```diff
@@ -373,6 +373,7 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                              ensure_ascii=self.ensure_ascii,
                              allow_nan=not self.strict) + '\n'
 
+
```

Comment:
if this styling convention isn't consistent with the rest of Doccano code, may be worth removing these